jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -844,3 +847,16 @@ function configure_kafka_jbpm_emitter(){
   fi
 
 }
+
+function configure_kie_executor_mdb(){
+  log_info "Configuring KieServerExecutorMfDB Max Sessions on ejb-jar.xml"
+  sed -i  's#<activation-config-property-value>queue/KIE.SERVER.EXECUTOR</activation-config-property-value>#<activation-config-property-value>queue/KIE.SERVER.EXECUTOR</activation-config-property-value>\
```

Comment:
do not do a sed like this.
edit the https://github.com/jboss-container-images/jboss-kie-modules/blob/main/jboss-kie-kieserver/added/WEB-INF/ejb-jar.xml#L51

and add the max session property there and 'sed' it with a default value.

Reply:
Ok

Reply:
didn't see the changes requested here, still the same.